### PR TITLE
DRUP-525: list our roadmap

### PR DESCRIPTION
**Pull request by cnovak**
Add info on project status and roadmap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+* __Debug__: enables administrators to configure and manage Apigee debug logs.
+* __Teams__: (Experimental) enables developers to be organized into teams.
+
+For more details read the [Apigee Edge module documentation](https://www.drupal.org/docs/8/modules/apigee-edge). 
+
+## Module status
+
+The core functionality of this module is complete and we are currently completing final testing and critical
+issue fixes. We encourage you to download and evaluate the module, and to use our 
+[Github issue queue](https://github.com/apigee/apigee-edge-drupal/issues) to give feedback, ask questions,
```
Comment (earth2marsh) [inline README.md]: `GitHub`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,36 @@
-Apigee Edge Drupal module
----
+# Apigee Edge Drupal module
 
-The Apigee Edge module enables you to integrate Drupal 8 with Apigee Edge.
+The Apigee Edge module enables you to integrate Drupal 8 with Apigee Edge.  
 
-**Note**: The Apigee Edge module requires Drupal 8.6.x or higher and PHP 7.1 or higher.
+* Users that register or are added to the Drupal site will be added as developers in Apigee Edge
+* Click on "My Apps" in user menu to get keys for Apigee Edge APIs
```
Comment (earth2marsh) [inline README.md]: I would have thought `My Apps` was for consumers to get their keys for the provider's APIs. Is this also how you get your keys for Apigee Edge APIs?

File: README.md [hunk 3]
```diff
@@ -41,21 +66,21 @@ module.**
...
-### Troubleshooting
+## Troubleshooting
 
 * **[File entity](https://www.drupal.org/project/file_entity) module.** If you installed the File entity module then you are going to need the latest patch from [this issue](https://www.drupal.org/project/file_entity/issues/2977747) otherwise you can run into some problems.
 * **[Key](https://www.drupal.org/project/key) module.** If you are using OAuth then you are going to need the latest patch from [this issue](https://www.drupal.org/project/key/issues/2982124) otherwise you can run into some problems.
 
-### Development
+## Development
 
 Development is happening in our [GitHub repository](https://github.com/apigee/apigee-edge-drupal). The drupal.org issue queue is disabled, we use the [Github issue queue](https://github.com/apigee/apigee-edge-drupal/issues) to coordinate development.
```
Comment (earth2marsh) [inline README.md]: 2nd link should be `GitHub` as text. 
Suggest a semi-colon after `issue queue is disabled`.
Consider `Development is happening in` > `Development is coordinated in`.

File: README.md (hunk 2 above)
Comment (cnovak) [reply README.md]: Yes, updated

File: README.md (hunk 3 above)
Comment (cnovak) [reply README.md]: Got first two, but last issue is already done - it says "Development is happening in" as the first part of the sentence.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,36 @@
-Apigee Edge Drupal module
----
+# Apigee Edge Drupal module
 
-The Apigee Edge module enables you to integrate Drupal 8 with Apigee Edge.
+The Apigee Edge module enables you to integrate Drupal 8 with Apigee Edge.
```
Comment (cnovak) [inline README.md]:
```suggestion
The Apigee Edge module enables YOU to integrate Drupal 8 with Apigee Edge.  
```

File: README.md [hunk 5]
```diff
@@ -39,23 +64,30 @@ module.**
...
+## Troubleshooting
 
 * **[File entity](https://www.drupal.org/project/file_entity) module.** If you installed the File entity module then you are going to need the latest patch from [this issue](https://www.drupal.org/project/file_entity/issues/2977747) otherwise you can run into some problems.
 * **[Key](https://www.drupal.org/project/key) module.** If you are using OAuth then you are going to need the latest patch from [this issue](https://www.drupal.org/project/key/issues/2982124) otherwise you can run into some problems.
 
-### Development
+## Development
 
-Development is happening in our [GitHub repository](https://github.com/apigee/apigee-edge-drupal). The drupal.org issue queue is disabled, we use the [Github issue queue](https://github.com/apigee/apigee-edge-drupal/issues) to coordinate development.
+Development is happening in our [GitHub repository](https://github.com/apigee/apigee-edge-drupal). The drupal.org issue queue is disabled; we use the [GitHub issue queue](https://github.com/apigee/apigee-edge-drupal/issues) to coordinate development.
```
Comment (cnovak) [inline README.md]:
```suggestion
Development is coordinated in our [GitHub repository](https://github.com/apigee/apigee-edge-drupal). The drupal.org issue queue is disabled; we use the [GitHub issue queue](https://github.com/apigee/apigee-edge-drupal/issues) to coordinate development.
```

File: README.md [hunk 6]
```diff
@@ -39,23 +64,30 @@ module.**
...
 1. Click **Install**.
 
-**Note**: If you do not configure the connection between Drupal and Apigee Edge, you will not be able to register developers on the site and may cause other issues with Drupal core functions. If you do not plan to configure the connection between Drupal and Apigee Edge, you should uninstall the Apigee Edge module.
+## Notes
+
+* If you do not configure the connection between Drupal and Apigee Edge, you will not be able to register developers on
+  the site and may cause other issues with Drupal core functions. If you do not plan to configure the connection between 
+  Drupal and Apigee Edge, you should uninstall the Apigee Edge module.
+* When you first install the module, existing users in the Drupal site may not have a developer account in Apigee Edge.
+  You can synchronized
```
Comment (earth2marsh) [inline README.md]: fragment?

File: README.md [hunk 7]
```diff
@@ -1,11 +1,36 @@
...
+or log issues.
+
+Following the same concept of [Drupal 8 Experimental modules](https://www.drupal.org/core/experimental), 
+some submodules are marked as "Experimental". They are provided for evaluation and testing purposes but are still 
+considered to be in development. Experimental modules are included in the "Apigee (Experimental)" package
+ on the "Extend" page of a Drupal site (/admin/modules).
 
-### Installing
+# Planned features
+* __Team Roles__: Site admins can manage roles and permissions available for in teams submodule.
```
Comment (jbennett13) [inline README.md]:
```suggestion
* __Team Roles__: Site admins can manage roles and permissions available for developers in teams submodule.
```

File: README.md (hunk 6 above)
Comment (cnovak) [reply README.md]: Me bad at multitasking

**Comment by earth2marsh** [comment]
lgtm

**Review by earth2marsh — COMMENTED** [review]
LGTM

**Review by earth2marsh — APPROVED** [review]
LGTM (for real this time)

**Pull request merged**